commands.js

```diff
@@ -1352,8 +1352,11 @@ exports.commands = {
 
 	mn: 'modnote',
 	modnote: function (target, room, user, connection) {
+		var groupName = Config.groups[room.modchat].name || room.modchat;
+		
 		if (!target) return this.parse('/help modnote');
 		if ((user.locked || room.isMuted(user)) && !user.can('bypassall')) return this.errorReply("You cannot do this while unable to talk.");
```

Comment:
Maybe we should just run it through `canTalk` instead of all these separate checks.